package.json

```diff
@@ -17,9 +17,9 @@
     "mongodb-uri": "0.9.7",
     "optval": "1.0.1",
     "redis": "^2.6.5",
-    "request": "2.83.0",
+    "request": "2.88.0",
     "underscore": "1.7.0",
-    "unifiedpush-node-sender": "0.12.1",
+    "unifiedpush-node-sender": "0.16.0",
```

Comment:
I know this is just a minor version bump, but given the lib (ours) is a `0.` version are we sure there are no breaking changes between `0.12.1` to `0.16.0`?

Reply:
Hi @damienomurchu ,

Following a few points. 

* if it has break changes it should be 1.0.0 and not 0.16.0. ( it is our lib and we follow [sem version](https://semver.org/) definition )
* Is usage is covered [here](https://github.com/feedhenry/fh-mbaas-api/blob/433d65a4b5e1d3ff7200072f60751ee3de9f3934/test/test_fhpush.js#L13)
* Also, if you check [here](https://github.com/aerogear/aerogear-unifiedpush-nodejs-client/releases) in the releases of this lib no break changes can be found so far.

Reply:
last link says to me there's nothing to worry about re breaking changes between versions, so approving PR